data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -487,6 +518,37 @@ spec:
                 description: Platform is configuration for machine pool specific to
                   the platform.
                 properties:
+                  alibabacloud:
+                    description: AlibabaCloud is the configuration used when installing
+                      on Alibaba Cloud.
+                    properties:
+                      instanceType:
+                        description: InstanceType defines the ECS instance type. eg.
+                          ecs.xn4.small
+                        type: string
+                      systemDisk:
+                        description: SystemDisk defines the system disk for ECS instances
+                          in the machine pool.
+                        properties:
+                          category:
+                            description: Category defines the category of the disk.
+                            type: string
+                          size:
+                            description: Size defines the size of the disk in gibibytes
+                              (GiB).
+                            minimum: 20
```

Comment:
Suggest increasing the size to a minimum of 120 GB

Reply:
will change it later